projects/storefrontlib/src/cms-components/checkout/services/checkout-step.service.ts

```diff
@@ -57,6 +57,17 @@ export class CheckoutStepService {
     this.routingService.go(nextUrl);
   }
 
+  goToStepWithIndex(stepIndex: number, activatedRoute?: ActivatedRoute): void {
```

Comment:
if navigate to the current page, then navigator will do nothing. So, this function works well without pass `activatedRoute`.

Reply:
Thanks, I didn't know the navigator handled this case already.
No need to complexity the function then.  I'll remove the `activatedRoute` parameter.